wire-runtime/src/main/java/com/squareup/wire/RuntimeMessageAdapter.java

```diff
@@ -93,40 +92,47 @@
   private final Class<M> messageType;
   private final Class<Builder<M, B>> builderType;
```

Comment:
We should probably change all these to be `Class<B>`, `Constructor<B>`, etc. since `B` _is_ a `Builder<M, B>`. Follow up.